src/libraries/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -57,20 +57,17 @@
     <Compile Include="System\Text\Json\Nodes\JsonArray.cs" />
     <Compile Include="System\Text\Json\Nodes\JsonArray.IList.cs" />
     <Compile Include="System\Text\Json\Nodes\JsonNode.cs" />
-    <Compile Include="System\Text\Json\Nodes\JsonNode.Dynamic.cs" />
     <Compile Include="System\Text\Json\Nodes\JsonNode.Operators.cs" />
     <Compile Include="System\Text\Json\Nodes\JsonNode.Parse.cs" />
     <Compile Include="System\Text\Json\Nodes\JsonNode.To.cs" />
     <Compile Include="System\Text\Json\Nodes\JsonNodeOptions.cs" />
     <Compile Include="System\Text\Json\Nodes\JsonObject.cs" />
-    <Compile Include="System\Text\Json\Nodes\JsonObject.Dynamic.cs" />
     <Compile Include="System\Text\Json\Nodes\JsonObject.IDictionary.cs" />
     <Compile Include="System\Text\Json\Nodes\JsonValue.CreateOverloads.cs" />
     <Compile Include="System\Text\Json\Nodes\JsonValue.cs" />
     <Compile Include="System\Text\Json\Nodes\JsonValueNotTrimmable.cs" />
     <Compile Include="System\Text\Json\Nodes\JsonValueOfT.cs" />
     <Compile Include="System\Text\Json\Nodes\JsonValueTrimmable.cs" />
-    <Compile Include="System\Text\Json\Nodes\MetaDynamic.cs" />
```

Comment:
Can we also get rid of the System.Linq.Expressions reference later in the file?

Reply:
Yes. Thanks